.github/workflows/automatic_pull_request_review.yml

```diff
@@ -42,12 +42,12 @@ jobs:
       run: bundle exec pod install
 
     - name: Build and Test
-      run: bundle exec fastlane build_and_test
+      run: bundle exec fastlane buildAndTestLane
```

Comment:
I think we can remove the suffix `Lane`. You can try to run `bundle exec fastlane buildAndTest`. If it's ok, please remove all suffixes :D

Reply:
`bundle exec fastlane buildAndTest`
It works. Thank you 🙌 .

Reply:
@nkhanh44 It's time to update this PR :D

Reply:
I fixed it at https://github.com/nimblehq/ios-templates/pull/395/commits/f92a92b9406367940c03752b37d76935337fe6e7 :D.